### PR TITLE
Update CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by Scienfitz**

### CODE REVIEW

**Review by AVHopp — APPROVED** [review]

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -7,5 +7,11 @@
 - Alex Lee (EMD Electronics, Tempe, Arizona, USA) [Contact](mailto:alex.lee@emdgroup.com), [Github](https://github.com/galaxee87)
 
 ## Contributors
+- Daniel Weber (Merck KGaA, Darmstadt, Germany):
```
[the remark this answers is not in the document]
Comment (Scienfitz) [reply CONTRIBUTORS.md:10]: good point, in the future its prob going to be by contrib date (newest below) if we add some sort of bot, but honestly i dont remember the order of these existing ones

[the remark this answers is not in the document]
Comment (AVHopp) [reply CONTRIBUTORS.md:10]: Ok. Would still appreciate a comment here, but can also live without one. If @AdrianSosic agrees, feel free to just resolve.

[the remark this answers is not in the document]
Comment (AdrianSosic) [reply CONTRIBUTORS.md:10]: Ok, suggestion: How about simply going full transparency here?
1. For the authors, we can add are "Sorted by <whatever, e.g. last name>"
2. For the contributions, I can easily look up the dates from the history, sort them, and we can add "Sorted by date of contribution (earliest first)"

[the remark this answers is not in the document]
Comment (Scienfitz) [reply CONTRIBUTORS.md:10]: just to be clear, you want me to reorder the names and also write a comment explicitly as to how theyre ordered?

[the remark this answers is not in the document]
Comment (AVHopp) [reply CONTRIBUTORS.md:10]: For the authors, however, I would also be fine with the two of you being the first two authors since BayBE is still mainly your baby (it took me way to long to remember how to write that word properly :D) and only have that specific ordering for contributors if you prefer that option.
TL;DR: Just do whatever you want but write down what you do :)

[the remark this answers is not in the document]
Comment (Scienfitz) [reply CONTRIBUTORS.md:10]: tbh I wouldnt do anything, leave the authors as is and have the contributors according to this order which i think is roughly chronological. but then aain we plan to have a proper contributors management whats the point of overengineering here?

**Review by AdrianSosic — APPROVED** [review]
